numpy/core/tests/test_deprecations.py

```diff
@@ -533,3 +533,11 @@ def test_get_numeric_ops(self):
         # other tests.
         self.assert_deprecated(np.set_numeric_ops, kwargs={})
         assert_raises(ValueError, np.set_numeric_ops, add='abc')
+
+
+class TestShape1Fields(_DeprecationTestCase):
+    warning_cls = FutureWarning
```

Comment:
I assume we never made a FutureDeprecationTestCase...

Reply:
grepping doesn't give anything